python/lsst/analysis/tools/actions/vector/calcRhoStatistics.py

```diff
@@ -0,0 +1,649 @@
+    leakage due to errors in PSF modeling to the weak lensing shear correlation
+    functions. The standard rho statistics are indexed from 1 to 5, and
+    this action calculates a sixth rho statistic, indexed 0.
+
+    Notes
+    -----
+    The exact definitions of rho statistics as defined in [1]_ are given below.
+    In addition to these five, we also compute the auto-correlation function of
+    the fractional size residuals and call it as the :math:`\rho_0( \theta )`.
+
+    .. math::
+
+       \rho_0(\theta) &= \left\langle \frac{\delta T_{PSF}}{T_{PSF}}(x) \frac{\delta T_{PSF}}{T_{PSF}}(x+\theta) \right\rangle  # noqa: E501, W505
+
+       \rho_1(\theta) &= \langle \delta e^*_{PSF}(x) \delta e_{PSF}(x+\theta) \rangle  # noqa: W505
+
+       \rho_2(\theta) &= \langle e^*_{PSF}(x) \delta e_{PSF}(x+\theta) \rangle
+
+       \rho_3(\theta) &= \left\langle (e^*_{PSF}\frac{\delta T_{PSF}}{T_{PSF}}(x)) \delta e_{PSF}(x+\theta) \right\rangle  # noqa: E501, W505
+
+       \rho_4(\theta) &= \left\langle (\delta e^*_{PSF}(x) (e_{PSF}\frac{\delta T_{PSF}}{T_{PSF}}(x+\theta)) \right\rangle  # noqa: E501, W505
+
+       \rho_5(\theta) &= \left\langle (e^*_{PSF}(x) (e_{PSF}\frac{\delta T_{PSF}}{T_{PSF}}(x+\theta)) \right\rangle  # noqa: E501, W505
+
+    The definition of ellipticity used in [1]_ correspond to ``epsilon``-type ellipticity, which is typically
+    smaller by a factor of 4 than using ``chi``-type ellipticity.
+
+    References
+    ----------
+    .. [1] Jarvis, M., Sheldon, E., Zuntz, J., Kacprzak, T., Bridle, S. L., et. al (2016).  # noqa: W501
+           The DES Science Verification weak lensing shear catalogues.
+           MNRAS, 460, 2245–2281.
+           https://doi.org/10.1093/mnras/stw990;
+           https://arxiv.org/abs/1507.05603
+    """
+
+    colRa = Field[str](doc="RA column", default="coord_ra")
+
+    colDec = Field[str](doc="Dec column", default="coord_dec")
+
+    colXx = Field[str](doc="The column name to get the xx shape component from.", default="{band}_ixx")
+
+    colYy = Field[str](doc="The column name to get the yy shape component from.", default="{band}_iyy")
+
+    colXy = Field[str](doc="The column name to get the xy shape component from.", default="{band}_ixy")
+
+    colPsfXx = Field[str](
+        doc="The column name to get the PSF xx shape component from.", default="{band}_ixxPSF"
+    )
+
+    colPsfYy = Field[str](
+        doc="The column name to get the PSF yy shape component from.", default="{band}_iyyPSF"
+    )
+
+    colPsfXy = Field[str](
+        doc="The column name to get the PSF xy shape component from.", default="{band}_ixyPSF"
+    )
+
+    ellipticityType = ChoiceField[str](
+        doc="The type of ellipticity to calculate",
+        allowed={
+            "chi": "Distortion, defined as (Ixx - Iyy)/(Ixx + Iyy)",
+            "epsilon": ("Shear, defined as (Ixx - Iyy)/" "(Ixx + Iyy + 2*sqrt(Ixx*Iyy - Ixy**2))"),
```

Comment:
That's not actually the "definition" of shear.  That's just one way to estimate shear.  Shear is defined by the elements in the 2x2 transformation matrix.

Reply:
Also, as per comment above, I'd prefer the "type" names be distortion and shear, rather than semi-arbitrary Greek letters.

Reply:
Just adding a vote in for understandable names not random greek letters. Remember that non WL people will see this, like me, and it would be great if it was understandable.